calender.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div>
+    <h1>Upcoming Events</h1>
+    <br>
+    <ul>
+      <li v-for="event in events" :key="event.id">
+        <div class = "eventSummary">{{ event.summary }}</div>
+        <div class = "eventTime" v-if="event.start && event.end">
+          {{ formatEventTime(event.start.dateTime || event.start.date, event.end.dateTime || event.end.date) }}
+        </div>
```

Comment:
I would recommend you abstract this into an `Event.vue` component, and create use them in an `EventList.vue` component.